.github/workflows/ci.yml

```diff
@@ -343,12 +404,77 @@ jobs:
           # microvmi_x.x.x_amd64.deb
           path: target/debian/*
 
+  build_windows:
+    # build libmicrovmi with all windows drivers
+    # and upload it as an artefact
+    needs: [virtualbox_windows, memflow]
+    runs-on: windows-2022
+    steps:
+      - name: install stable toolchain with clippy
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          components: clippy
+
+      - name: Clone Icebox repository
+        uses: actions/checkout@v2
+        with:
+          repository: thalium/icebox
+          path: icebox
+
+      - uses: actions/checkout@v2
+        with:
+          path: libmicrovmi
+
+      # adds cl.exe in the PATH
+      - name: Setup command prompt for Microsoft Visual C++
+        uses: ilammy/msvc-dev-cmd@v1
+
+      - name: Compile and install FDP library
+        run: |
+          # install in C:\FDP to avoid issues with quotes etc
+          cl /c FDP.cpp /out:FDP.obj
+          cl /LD FDP.obj /link /out:FDP.dll
+          lib /NOLOGO FDP.obj /OUT:FDP.lib
+          dir
+          New-Item -Path "C:\FDP" -ItemType "directory"
+          copy FDP.dll "C:\FDP"
+          copy FDP.lib "C:\FDP"
+          copy "include\*" "C:\FDP"
+        working-directory: icebox/src/FDP
+
+      - name: Install Clang
+        run: choco install llvm
+
+      - name: build libmicrovmi with virtualbox and memflow driver
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
```

Comment:
Since you are planning to actually release the artifacts generated by this command, wouldn't it make more sense to issue a release build here instead of a debug build?

Reply:
I was planning to make another PR where I switch the build from debug to release, while keeping the debug symbols.